cvat/apps/cf/README.md

```diff
@@ -0,0 +1,12 @@
+## Git Integration For Annotation Storage
```

Comment:
Git Integration For ...???

Reply:
@nmanovic I prematurely included the CF app I've been building and this readme was associated with it.  It's gone now. Apologies!